python/tvm/relay/op/transform.py

```diff
@@ -1322,6 +1322,20 @@ def adv_index(inputs):
     return _make.adv_index(Tuple(inputs))
 
 
+def sparse_fill_empty_rows(sparse_indices, sparse_values, dense_shape, default_value):
+    """
+    This op exactly follows the documentation here:
+    https://www.tensorflow.org/api_docs/python/tf/sparse/fill_empty_rows
```

Comment:
Instead of linking to the tensorflow documentation, could you write out what this op does. Please include parameters, return values, and examples.

Reply:
Sure.